hadoop-ozone/dev-support/checks/acceptance.sh

```diff
@@ -17,19 +17,32 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd "$DIR/../../.." || exit 1
 
 REPORT_DIR=${OUTPUT_DIR:-"$DIR/../../../target/acceptance"}
-mkdir -p "$REPORT_DIR"
 
 OZONE_VERSION=$(grep "<ozone.version>" "pom.xml" | sed 's/<[^>]*>//g'|  sed 's/^[ \t]*//')
 DIST_DIR="$DIR/../../dist/target/ozone-$OZONE_VERSION"
 
 if [ ! -d "$DIST_DIR" ]; then
     echo "Distribution dir is missing. Doing a full build"
-    "$DIR/build.sh"
+    "$DIR/build.sh" -Pjacoco
 fi
 
+mkdir -p "$REPORT_DIR"
+
+export HADOOP_OPTS='-javaagent:/opt/hadoop/share/jacoco/jacoco-agent.jar=destfile=/tmp/jacoco.exec,includes=org.apache.hadoop.ozone.*:org.apache.hadoop.hdds'
```

Comment:
Now `acceptance.sh` requires `jacoco-agent.jar`, which is only built when invoked with `-Pjacoco`.  I would like to suggest moving this definition to the GitHub Actions definition (in the `env` section, where `KEEP_IMAGE` is defined, too).

Reply:
I think this is fixed. I prefer to keep the logic in the shell scripts to keep our build as github independent as possible, but with the improved approach this is done (ENV in the github file, logic in the script)